README.md

```diff
@@ -1,2 +1,168 @@
-# spend-tracker
-Spend Tracker is a highly intuitive and dynamic app that lets users record their expenses. Users can sync their data across multiple devices and protect it with a password. Users can also customize their categories, and themes to suit their preferences.
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./public/logo.svg" alt="logo" width="256"  height="auto" />
+  <br/>
+
+  <h3><b>SPEND TRACKER</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Spend Tracker <a name="about-project"></a>
+
+**Spend Tracker** is a fantastic app that helps you manage your money and budget wisely. With **Spend Tracker**, you can easily track your income and expenses, set goals and limits, and see your progress in real time. **Spend Tracker** also gives you insights and tips on how to save more and spend less. Whether you want to pay off debt, save for a vacation, or just have more control over your finances, **Spend Tracker** is the perfect tool for you. Download **Spend Tracker** today and start your journey to financial freedom!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **User can create categories.**
+- **User can create transactions.**
+- **User can see the total amount expended on each category.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://pss-spend-tracker.onrender.com)
+
+- [Live Presentation Link](https://www.loom.com/share/d8036b29844640608267d17e9eaf1012)
```

Comment:
- [ ] `required`
- [ ] Amazing work so far 👏 
- [ ] However I'm getting an error when I try to add something to the catergory
- [ ] please see screenshot below

![image](https://user-images.githubusercontent.com/66230306/235327301-856100ec-f92c-4a97-85d9-adc7e2bf5d16.png)